web/src/components/theme.js

```diff
@@ -55,6 +55,13 @@ export const darkTheme = {
         },
       },
     },
+    MuiPaper: {
+      styleOverrides: {
+        root: {
+          backgroundImage: "none",
+        },
+      },
+    },
   },
```

Comment:
Can you explain this? Where do we have a background image that would get in the way?

Reply:
will add a comment - there was an odd greyish background being added by MUI to the sidebar on narrow screens

| before | after |
| --- |  --- |
| ![before](https://github.com/binwiederhier/ntfy/assets/132819643/fb1a79cb-91ca-469e-ba29-b53e676ed0b6) | ![after](https://github.com/binwiederhier/ntfy/assets/132819643/0d656019-6058-42da-8458-9308ab5c0328) |